framework/src/application.ts

```diff
@@ -215,7 +209,7 @@ export class Application {
 		this.logger.info(
 			'Contribution guidelines can be found at Lisk-docs: https://github.com/LiskHQ/lisk-docs/blob/build/CONTRIBUTING.adoc',
```

Comment:
Checked accidentally, seems this link is wrong. Its referring to contribution to the documentation not the sdk. SDK contribution document is https://github.com/LiskHQ/lisk-sdk/blob/development/docs/CONTRIBUTING.md